examples/js/ViveController.js

```diff
@@ -1,47 +1,95 @@
 /**
  * @author mrdoob / http://mrdoob.com
+ * @author stewdio / http://stewd.io
  */
 
-THREE.ViveController = function ( id ) {
+THREE.ViveController = function( id ) {
 
 	THREE.Object3D.call( this );
 
-	var scope = this;
-	var gamepad;
+	var gamepad, scope = this;
 
-	this.getGamepad = function () { return gamepad; };
+	this.getGamepad = function() {
+
+		return gamepad;
+
+	}
 	this.matrixAutoUpdate = false;
 	this.standingMatrix = new THREE.Matrix4();
+	this.axes = [ 0, 0 ];
+	this.thumbpadIsPressed = false;
+	this.triggerIsPressed = false;
+	this.gripsArePressed = false;
+	this.menuIsPressed = false;
```

Comment:
Are you creating all these variables publicly because you access them from the outside? I think this is the only thing about this PR that I can see bringing problems in the future.

Is it ok if I turn them into private for now?